build/lib/release.sh

```diff
@@ -434,7 +434,7 @@ function kube::release::package_test_tarball() {
   # Add the test image files
   mkdir -p "${release_stage}/test/images"
   cp -fR "${KUBE_ROOT}/test/images" "${release_stage}/test/"
-  tar c ${KUBE_TEST_PORTABLE[@]} | tar x -C ${release_stage}
+  tar c "${KUBE_TEST_PORTABLE[@]}" | tar x -C "${release_stage}"
```

Comment:
I think this might cause issues too.

Reply:
It shouldn't, since you are expanding the array with "$xx{@}"

Reply:
You can verify that this works with something like this script:

```bash
#!/bin/bash

# create a test directory and cd there
mkdir /tmp/test_dir
cd /tmp/test_dir

# test variables
release_stage='test dir'
readonly KUBE_TEST_PORTABLE=(
  foo
  bar
  'x x'
)

# prepare the target directory and files
mkdir "${release_stage}"
touch foo bar 'x x'

# test command
tar c "${KUBE_TEST_PORTABLE[@]}" | tar x -C "${release_stage}"

# should contain foo, bar, and 'x x'
ls "${release_stage}"
```